src/lib.rs

```diff
@@ -687,3 +687,19 @@ impl BlitPipeline {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    fn assert_is_send<T: Send>() {}
+
+    #[test]
+    fn check_scene_is_send() {
+        assert_is_send::<super::Scene>();
+    }
+
+    #[test]
+    #[cfg(all(feature = "wgpu", not(target_arch = "wasm32")))]
```

Comment:
Can we have a comment explaining why this fails on wasm?

Reply:
Sure, but I don't know a clear place to point to that adequately describes the status.

There is https://github.com/gfx-rs/wgpu/discussions/4127 ... but some of the discussion is happening in the WebGPU groups (about what guarantees should be provided by the browsers).

Fundamentally, right now, `wgpu` types aren't `Send`, but can be forced to be with an extra feature flag:

```
## Implement `Send` and `Sync` on Wasm, but only if atomics are not enabled.
##
## WebGL/WebGPU objects can not be shared between threads.
## However, it can be useful to artificially mark them as `Send` and `Sync`
## anyways to make it easier to write cross-platform code.
## This is technically *very* unsafe in a multithreaded environment,
## but on a wasm binary compiled without atomics we know we are definitely
## not in a multithreaded environment.
fragile-send-sync-non-atomic-wasm = [
    "hal/fragile-send-sync-non-atomic-wasm",
    "wgc/fragile-send-sync-non-atomic-wasm",
    "wgt/fragile-send-sync-non-atomic-wasm",
]
```

Reply:
Right, this is probably fine then

Reply:
I've added a comment.